DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: RedditExtractoR
 Type: Package
 Title: Reddit Data Extraction Toolkit
-Version: 3.0.5
+Version: 3.0.6
```

Comment:
We also need to update the date field on line 10 in this file, just set that to today's date otherwise CRAN will complain

Reply:
Oh yeah, duh, sorry - that was obvious! Commit 5ad5cad on my fork has this updated. But I've never been involved in a workflow before so no idea how to merge it to this PR. I've googled and clicked everything but can't seem to work it out sorry.

Reply:
Hang on, it seems like it has done it automatically. That's pretty clever.